### ISSUE TITLE
Add option 'has been removed'

### ISSUE THREAD

**Issue by PanderMusubi** [westnordost/StreetComplete#1097]
As, in the Netherlands, post boxes (to send mail) are disappearing everywhere. When a quest asks for collection times, an option 'has been removed' is a viable one.

Perhaps a query per county could indicate which kind of SC supported objects are disappearing with a certain high rate in order to add that 'has been removed' option is a practical addition to a query in that country or region.

Phone boots and post offices have disappeared all together here, but that might be an ongoing process in another country, so will more of certain objects that get outdated.

**Comment by westnordost** [comment]
In that case, you ought to leave a note. It is not possible to delete elements with StreetComplete.

**Comment by ENT8R** [comment]
What speaks against creating a note for that purpose? IMO notes should be used in those cases because if it gets removed automatically by StreetComplete in the background and nobody notices it (maybe in a region where not so many mappers are active) there is a chance that somebody just didn't found it and removes it without any noise...
Also SC does not remove any elements at all, but only modifies existing ones...

**Comment by PanderMusubi** [comment]
Notes are also hard to query as different language and wordings are used when some object has been removed.

**Comment by PanderMusubi** [comment]
Maybe 'needs review' should be added.

**Comment by westnordost** [comment]
Needs review is for a different purpose.
Review ≠ Resurvey

**Comment by PanderMusubi** [comment]
Does tag 'needs resurvey' exist?

**Comment by matkoniecz** [comment]
It would be very useful to have option like this that would leave a prewritten note.

At this moment I need to spam "this object is gone" constantly.

It is also not obvious for new users that one should leave note in that case.

StreetComplete removing objects is not feasible (I written a longer explanation why in one of duplicates of this issue).

**Comment by rugk** [comment]
> It would be very useful to have option like this that would leave a prewritten note.

So when the note popup is shown just suggest some common text? That sounds good, IMHO. Users can add it with a few clicks, can adjust it/add more information and send the result.

**Comment by matkoniecz** [comment]
I thought about silently leaving note, like maps.me is doing but it may be better.

**Comment by PanderMusubi** [comment]
Is maps.me leaving a note containing a preformatted message? If we used the same, it will be picked up by the same filters (if there are any). Also, leaving a preformatted message in English, makes using a query to collect them all easy.

**Comment by matkoniecz** [comment]
> Is maps.me leaving a note containing a preformatted message?

Yes, it includes some info allowing to identify object and optional part from user.

**Comment by westnordost** [comment]
I am not convinced that marking things as deleted (via a note) should be implemented as a "Other answer..." option. "Forcing" the user to leave a note in that case gives him the chance to explain what perhaps in it's stead is there now plus add a photo. A Maps.Me-like staple "has been deleted" note is not that helpful because at least in the case of Maps.Me, it usually requires a resurvey by some other user to acknowledge this.
A custom-written "has been gone, xyz is there now / it's now vacant but sign is still up etc" respond usually has enough credibility (especially if a photo is attached) that usually people who solve these notes will do that without further need for confirmation.

**Comment by PanderMusubi** [comment]
So it depends on what has gone. If a business has gone away, the building indeed stays and might have another business using it.

Something like a post box, recycle bin or phone booth that has been removed, doesn't leave something else in place. Uploading a 3 MB picture of what isn't there seems a bit too much, IMHO. Especially when such objects are being removed 'en masse'.

Perhaps a mixed note could be used. Starting with something which is to to be queried and optionally a custom message after it. What do you think of that?
